lib/ui/src/modules/ui/components/stories_panel/stories_tree/tree_decorators.js

```diff
@@ -2,35 +2,50 @@ import { decorators } from 'react-treebeard';
 import { Icons } from '@storybook/components';
 import React from 'react';
 import PropTypes from 'prop-types';
+import { withCSSContext } from '@emotion/core';
 import { MenuLink } from '../../../containers/routed_link';
 import MenuItem from '../../menu_item';
 import treeNodeTypes from './tree_node_type';
 import { highlightNode } from './tree_decorators_utils';
 
 function noop() {}
 
-function ToggleDecorator({ style }) {
+function ToggleDecorator({ style, theme }) {
   const { height, width, arrow } = style;
+  const { treeArrow } = theme;
```

Comment:
This looks messy, but it doesn't matter too much, since that's all getting cleanup up with the UI overhaul.

Reply:
Sure. Lets cleanup with UI overhaul